deployments/examples/placements/README.md

```diff
@@ -20,13 +20,22 @@
 App placements rules in Yunikorn contains `Provided Rule`, `User Name Rule`, `Fixed Rule`, `Tag Rule`.
 Every placement example includes a example yaml file and a config yaml file.
 The sleep pod is described in the example file and the partition is provided in the config yaml file.
-Before deploying the pods, the configuration field in yunikorn-release/helm/yunikorn/value.yaml should be replaced by the configuration field in the config.yaml.
 
 * [Provided Rule](./provided)
 * [User Name Rule](./username)
 * [Fixed Rule](./fixed)
 * [Tag Rule](./tag)
 
+## How to adopt the queues in `config.yaml`
+Before deploying the pods, threre are some modifications required in [release/value.yaml](yunikorn-release/helm/yunikorn/value.yaml).
```

Comment:
Typo here - this should be `values.yaml`

Reply:
Oops,  I typed wrong words here.
I am going to update this words including `values.yaml` and `queues.yaml`.